src/Perpetuum/Services/Mail/MailHandler.cs

```diff
@@ -185,5 +185,39 @@ public static int NewMailCount(Character character)
         {
             return Db.Query().CommandText("newMailCount").SetParameter("@characterID", character.Id).ExecuteScalar<int>();
         }
+
+        public static ErrorCodes SendWelcomeMailBeginTutorial(Character newPlayer)
+        {
+            return SendWelcomeMail(newPlayer, 1);
+        }
+
+        public static ErrorCodes SendWelcomeMailExitTutorial(Character newPlayer)
+        {
+            return SendWelcomeMail(newPlayer, 2);
+        }
+
+        private static ErrorCodes SendWelcomeMail(Character newPlayer, int mailId)
+        {
+            // TODO: Change the sender character name for something more/less appropriate
+            Character sender = Character.GetByNick("overlord");
+
+            string subject, body;
+            using (var scope = Db.CreateTransaction())
+            {
+                var records = Db.Query()
+                    .CommandText("select subject, body from premademail where id = @mailId")
+                    .SetParameter("@mailId", mailId.ToString())
+                    .Execute();
+                subject = records.First().GetValue<string>(0);
```

Comment:
Also here you should use the column names when parsing the GetValue
`record.GetValue<string>("subject");`